### PR TITLE
Show all samples button for sample view

### PR DESCRIPTION
**Pull request by inodb**
Fix https://github.com/cBioPortal/cbioportal/issues/2510

screen shot 2017-06-09 at 4 17 39 pm

### CODE REVIEW

**Comment by inodb** [comment]
@alisman Might be good here to show # of other samples available as well, or maybe as you suggested just highlight the particular sample we selected and update components accordingly (also feels a little more single appy). This at least fixes basic requirement tho

File: src/pages/patientView/PatientViewPage.tsx [hunk 1]
```diff
@@ -183,6 +183,13 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
                     </div>
                 );
             });
+
+            if (sampleHeader && sampleHeader.length > 0 && patientViewPageStore.pageMode === 'sample' && patientViewPageStore.patientId) {
+                debugger;
+                sampleHeader = sampleHeader.concat(
```
Comment (onursumer) [inline src/pages/patientView/PatientViewPage.tsx]: why not just `sampleHeader.push(...)`?

File: src/pages/patientView/PatientViewPage.tsx [hunk 2]
```diff
@@ -183,6 +183,13 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
                     </div>
                 );
             });
+
+            if (sampleHeader && sampleHeader.length > 0 && patientViewPageStore.pageMode === 'sample' && patientViewPageStore.patientId) {
+                debugger;
```
Comment (onursumer) [inline src/pages/patientView/PatientViewPage.tsx]: I guess we don't need this `debugger` :)

**Comment by inodb** [comment]
@onursumer thanks for the review, will update!

**Pull request merged**